extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfigBuilder.java

```diff
@@ -662,6 +663,9 @@ public OidcTenantConfigBuilder provider(Provider provider) {
      * @return build {@link io.quarkus.oidc.OidcTenantConfig}
      */
     public io.quarkus.oidc.OidcTenantConfig build() {
+        if (tenantId.isEmpty()) {
```

Comment:
Would it make sense to have a dedicated PR for this fix as it won't only apply to this PR ?

Reply:
`OidcTenantConfigBuilder` is only in the main and in no released version, so I think it is not necessary. (unless I misunderstood it, this change is about allowing users to create default tenants without setting default tenant id explicitly, which I strongly suspect most of the users don't know where to find).